modules/op-about-taskrun.adoc

```diff
@@ -0,0 +1,38 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="about-taskrun_{context}"]
+= About TaskRun
+
+A _TaskRun_ instantiates a Task for execution with specific inputs, outputs, and execution parameters on a cluster. It can be invoked on its own or as part of a Pipeline.
```

Comment:
@vdemeester Isn't `...as part of a PipelineRun` more precise?

Reply:
Yeah, as we are talking about `TaskRun`, I would use `PipelineRun` here.